rollbar-reactive-streams/src/main/java/com/rollbar/reactivestreams/notifier/config/ConfigBuilder.java

```diff
@@ -461,6 +469,32 @@ public ConfigBuilder truncateLargePayloads(boolean truncate) {
     return this;
   }
 
+  /**
+   * <p>
+   * Maximum Telemetry Events sent in a payload. Default is
+   * {@value RollbarTelemetryEventTracker#MAXIMUM_CAPACITY_FOR_TELEMETRY_EVENTS}.
+   * </p>
```

Comment:
Could you update the doc to explain that setting a custom tracker causes the value set here to be ignored?